core/tests/protractor_utils/editor.js

```diff
@@ -377,9 +386,10 @@ var addParameterChange = function(paramName, paramValue) {
 // should be specified after the ruleName as additional arguments. For example
 // with interaction 'NumericInput' and rule 'Equals' then there is a single
 // parameter which the given answer is required to equal.
-var _selectRule = function(ruleElement, interactionId, ruleName) {
+var _selectRule = function(ruleElement, interactionId, ruleName,
+  skipParameters) {
```

Comment:
@seanlip,
I  run stateEditor e2e tests, unfortunately didn't run others, I assumed that _selectRule is only used in this class. What caused you to surprise?